serveradmin/serverdb/migrations/0005_attribute_clone.py

```diff
@@ -10,11 +10,6 @@ class Migration(migrations.Migration):
     ]
 
     operations = [
-        migrations.AddField(
```

Comment:
Was that field somewhere added before now? Because I could not find it and this migration would then fail, because of the RunSQL afterwards.

Reply:
It just moved to the initial migration.